apps/wallet-ui/src/lib/chains.ts

```diff
@@ -0,0 +1,31 @@
+import type { ChainMetadata } from "thirdweb/chains";
```

Comment:
probably add `import "server-only";` since I assume this will only be called as server action / on backend?